content/manuals/desktop/features/gordon.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Ask Gordon
+description: Learn how to streamline your workflow with Docker's AI-powered assistant.
+weight: 10
+params:
+  sidebar:
+    badge:
+      color: blue
+      text: Beta
+---
+
+{{% restricted title=Beta %}}
+Ask Gordon is a [Beta](/manuals/release-lifecycle.md) feature, and only members
+of the Ask Gordon beta program can access it. Features, user interface, and
+behavior are subject to change in future releases.
+
+{{< button text="Apply for access" url="https://docker.qualtrics.com/jfe/form/SV_dmVHFjQ4fZlrEOy" >}}
```

Comment:
Suggestion: Should the Qualtrics link open in a new tab?

Reply:
No, this is a UX antipattern; I learned about it recently from #21291